wnprc_billing/resources/web/wnprc_billing/form/field/ChargeItemField.js

```diff
@@ -0,0 +1,36 @@
+
+Ext4.define('WNPRC_Billing.form.field.ChargeItemField', {
+    extend: 'EHR_Billing.form.field.EHRBillingRowObserverEntryField',
+    alias: 'widget.wnprc_billing-chargeitemfield',
+
+    initComponent: function(){
+
+        this.addListener(
+        {
+                scope:this,
+                select: function (combo, recs) {
+
+                    var chargeDate = EHR.DataEntryUtils.getSiblingValue(combo, 'date'); // get date value from data entry form
+                    var chargeId = recs[0].get('rowId');
+
+                    LABKEY.Query.selectRows({
+                        schemaName: 'ehr_billing_public',
+                        queryName: 'chargeRates',
+                        filterArray: [
+                            LABKEY.Filter.create('chargeId', chargeId, LABKEY.Filter.Types.EQUAL),
+                            LABKEY.Filter.create('startDate', chargeDate.format("Y-m-d"), LABKEY.Filter.Types.DATE_LESS_THAN_OR_EQUAL),
+                            LABKEY.Filter.create('endDate', chargeDate.format("Y-m-d"), LABKEY.Filter.Types.DATE_GREATER_THAN_OR_EQUAL)
+                        ],
+                        columns: 'chargeId, startDate, endDate, unitCost',
```

Comment:
Small thing, but I don't think startDate and endDate would be needed in the columns for the response. They are being filtered on but it looks like the response only uses the unitCost value.

Reply:
You are right, now changed.